scripts/convert_publish.py

```diff
@@ -146,7 +146,8 @@ def convert_posts(self, query = {}):
                     print("messages converted -- ", added)
                     self.publish_tables.writeCache()
                 added += 1
-                utils.printProgressBar(passed + 1, length, prefix = 'Progress:', suffix = 'Complete', length = 50)
+                if (passed % utils.recording_step == 0 or passed < utils.recording_step):
```

Comment:
What about case when `passed < utils.recording_step`? Do we need to print a progress on each object in this case?
Why so difficultly? Why not just move the line a little higher and do it under condition `if added % self.cache_period == 0:`?